src/control/ioserver/exec.go

```diff
@@ -0,0 +1,39 @@
+// +build linux
+
+package ioserver
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/pkg/errors"
+
+	"github.com/daos-stack/daos/src/control/log"
+)
+
+const binaryName = "daos_io_server"
+
+func (srv *Instance) run(ctx context.Context, args, env []string) error {
+	binaryPath, err := exec.LookPath(binaryName)
+	if err != nil {
+		return errors.Wrapf(err, "Unable to find %s in path", binaryName)
+	}
+
+	cmd := exec.CommandContext(ctx, binaryPath, args...)
+	// TODO: Configure stdout/stderr to log
```

Comment:
have just changed this in https://github.com/daos-stack/daos/pull/569

Reply:
In the long run, I think all logging from the control plane and the managed ioserver instances should go to the same place, and it should be configurable to a file, syslog, or stdout. In the systemd/containerized world, stdout is preferred.